installbuilder/datafiles/Base_DSC.data

```diff
@@ -169,14 +169,16 @@ EOF
 
 chmod u+x $tmpfile
 $tmpfile 1> /dev/null 2> /dev/null
-if [ $? -ne 0 ]; then
+tmpfile_ctypes=$?
+rm $tmpfile
+
+python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
@lagalbra if script is not using /tmp path then this will probably not solve the problem customer is facing. And in that it will be very helpful if we update the error message to indicate which check failed.